doc/guides/CustomStartupGuide.md

```diff
@@ -529,6 +532,7 @@ Follow these instructions to update devices.
    deviceName: device-onvif-camera
    ```
    >NOTE: The device with name `device-onvif-camera` is a stand-in device and can be ignored.  
+   >NOTE: The deviceName `Camera001` will be used in all the following commands.  Please use the exact deviceName obtained for your device.  
    >NOTE: The `jq -r` option is used to reduce the size of the displayed response. The entire device with all information can be seen by removing `-r '"deviceName: " + '.devices[].name'', and replacing it with '.'`
 
```

Comment:
Since there are so many notes in this section, I wonder if there is a way we could combine them or streamline this better.

Reply:
I have removed the 'Camera001' note.  Would that be sufficient or should I look at streamlining even further?

Reply:
This works.